src/Arcus.Observability.Telemetry.Serilog.Filters/TelemetryTypeFilter.cs

```diff
@@ -74,31 +68,16 @@ public bool IsEnabled(LogEvent logEvent)
             switch (TelemetryType)
             {
                 case TelemetryType.Dependency:
-                    return IsFilteringRequired(ContextProperties.DependencyTracking.DependencyLogEntry, logEvent);
+                    return logEvent.Properties.ContainsKey(ContextProperties.DependencyTracking.DependencyLogEntry);
```

Comment:
Is this a deliberate change or because master was not merged because this would introduce a regression bug?

Reply:
Thought I was on the latest version of master, but I must have made a mistake somewhere. Reverted the changes to the file.

Reply:
No problem, can you merge master as well please?

Reply:
The branch is created from the master branch, I think the changes have something to do with me copying the files after I forked the project. When I pull and merge master I get the message `Already up to date.`.